package.json

```diff
@@ -51,6 +53,7 @@
         "bluebird": "3.7.2",
         "eslint": "6.8.0",
         "faucet": "0.0.1",
+        "get-port": "5.1.1",
```

Comment:
is this not a dev dependency?

Reply:
Yes it is. And it is under devDependencies.